### PR TITLE
Fix wrong Vitals-page to Russian translation

### PR DESCRIPTION
**Pull request by denisstasyev**
Fixes some errors in translation of the [Vitals-page](https://web.dev/vitals/)

### CODE REVIEW

**Review by malchata — APPROVED** [review]

File: src/site/content/ru/vitals/index.md [hunk 1]
```diff
@@ -13,19 +13,19 @@ tags:
   - web-vitals
 ---
 
-Оптимизация качества взаимодействия с пользователемключ к долгосрочному успеху любого сайта в Интернете. Web Vitals поможет владельцам бизнеса, маркетологам или разработчикам количественно оценить впечатления от сайта и определить возможности для улучшения.
+Оптимизация качества взаимодействия с пользователем - ключ к долгосрочному успеху любого сайта в Интернете. Web Vitals поможет владельцам бизнеса, маркетологам или разработчикам количественно оценить впечатления от сайта и определить возможности для улучшения.
 
 ## Обзор
```
Comment (lashaie08) [inline src/site/content/ru/vitals/index.md:18]: #7006

File: src/site/content/ru/vitals/index.md [hunk 2]
```diff
@@ -13,19 +13,19 @@ tags:
   - web-vitals
 ---
```
Comment (lashaie08) [inline src/site/content/ru/vitals/index.md:15]: >  @``

File: src/site/content/ru/vitals/index.md [hunk 3]
```diff
@@ -13,19 +13,19 @@ tags:
...
 
-Оптимизация качества взаимодействия с пользователемключ к долгосрочному успеху любого сайта в Интернете. Web Vitals поможет владельцам бизнеса, маркетологам или разработчикам количественно оценить впечатления от сайта и определить возможности для улучшения.
```
Comment (lashaie08) [inline src/site/content/ru/vitals/index.md:16]: пользователемключ #6566

**Review by lashaie08 — CHANGES_REQUESTED** [review]
[#6868]()

File: src/site/content/ru/vitals/index.md (hunk 3 above)
Comment (denisstasyev) [reply src/site/content/ru/vitals/index.md:16]: Thanks for the review, but could you clarify your comments?

**Pull request merged**